data/property-sort-orders/smacss.txt

```diff
@@ -121,41 +167,66 @@ color
 font
 font-family
 font-size
+font-size-adjust
+font-stretch
+font-effect
+font-smooth
 font-smoothing
+-moz-osx-font-smoothing
+-webkit-font-smoothing
```

Comment:
There're no other vendor prefixed rules in these lists, so I don't think it's a good idea to start adding these here.